OpenMTR/OpenMTR.csproj

```diff
@@ -58,6 +58,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Models\Meter.cs" />
+    <Compile Include="Code\OMTR.cs" />
```

Comment:
I'm guessing the file browser and other form elements were ninja'd in with the initial commit so I am putting this here. 

The modified image needs to be a little bigger.